materialize-sql/std_sql.go

```diff
@@ -61,27 +62,6 @@ func StdSQLExecStatements(ctx context.Context, db *sql.DB, statements []string)
 	return conn.Close() // Release to pool.
 }
 
-// StdIncrementFence increments the value for the fence for all checkpoints of the provided
-// materialization.
-func StdIncrementFence(ctx context.Context, db *sql.DB, ep *Endpoint, materialization string) error {
```

Comment:
Minor callout here: We won't need this anymore with runtime-driven applies since all materialization shards will be stopped when the apply happens, so there's no reason to fence off other instances.